pycrostates/cluster/tests/test_kmeans.py

```diff
@@ -799,6 +799,25 @@ def test_fit_with_bads(caplog):
     caplog.clear()
 
 
+def test_fit_picks():
+    raw = raw_meg.copy().pick_types(meg=True, eeg=True, eog=True)
+    ModK_ = ModKMeans(
+        n_clusters=n_clusters,
+        n_init=10,
+        max_iter=100,
+        tol=1e-4,
+        random_state=1,
+    )
+    with pytest.raises(ValueError):
+        ModK_.fit(raw, picks=None)  # fails -> MEG + EEG
+    with pytest.raises(ValueError):
+        ModK_.fit(raw, picks="meg")  # fails -> grad + mag
+    with pytest.raises(ValueError):
+        ModK_.fit(raw, picks="data")  # fails -> eeg + grad + mag
```

Comment:
Could you add `match="matching string"`, with maybe part of the string which is generated based on the channels?